app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -298,6 +300,11 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
         )
     }
 
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _bookmarkInteractor = null
```

Comment:
Why is this needed?

Reply:
Here's a second possible fix to the same leak: https://github.com/mozilla-mobile/fenix/compare/master...ekager:bookmark-snackbar-fix2

Maybe slightly preferable since I really don't think the interactor should have references to any views to begin with... But also always leaves the possibility that someone else adds a reference later and regresses that fix. The definitive nulling out of the interactor protects against that. 🤷‍♀️